### ISSUE TITLE
Version bump?

### ISSUE THREAD

**Issue by maclover7** [thoughtbot/paperclip#1848]
A lot of changes have been merged in lately -- does this warrant a version bump? @jyurek

**Comment by tute** [comment]
Jon said next version should be a minor version, not a patch version. We'll work on this.

**Comment by maclover7** [comment]
@tute @jyurek Does 4.3.0 sound good? When should we release -- Friday?

**Comment by tute** [comment]
Yes, @maclover7. AFAIK we are ready to release 4.3.0, and Friday sounds good. If you want to prepare a branch with the version bump and the CHANGELOG updated that will be very much welcomed. Thank you!

**Comment by maclover7** [comment]
Whoops @tute -- forgot about this. Does tomorrow or Wednesday work for releasing 4.3.0? Also, we'll need @jyurek to push up to RubyGems.org unless he adds one of us as contributors.

**Comment by tute** [comment]
Everyone at thoughtbot has access for these releases, I can technically do it. I'd wait on the green flag from @jyurek though, as I believe he wanted to merge a few changes before releasing. Jon, what changes were those?

Also, we need a green CI since it got red. This is embarrasing. If you want to investigate @maclover7 that will take us a meaningful step closer! https://github.com/thoughtbot/paperclip/issues/1863

**Comment by maclover7** [comment]
Was the build green when it was merged in? @tute

**Comment by tute** [comment]
Regarding CI, it looks green now, I closed the issue. I wonder if CI is not always green and sometimes fail.

Anyhow, you can start a branch preparing a release if you'd like @maclover7. @jyurek will be delighted. :)

**Comment by maclover7** [comment]
@tute So will we be requiring Ruby >= 2.0.0 going forward?

**Comment by maclover7** [comment]
#1875

**Comment by tute** [comment]
Closing in favor of PR.
